docs/notebooks/downstream_machine_learning_regression_evaluation.ipynb

```diff
@@ -0,0 +1,311 @@
+    " \n",
+    " To use the Gretel Evaluate Regression model, you must indicate the target column. Optionally, you can change the test-holdout amount, which is a float indicating the amount of real-world data you want to use as a holdout for testing the downstream regression models. You can also optionally select which models to use and which metric to optimize for. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#### SUPPORTED MODELS AND METRICS ####\n",
+    "## If you want to only use certain regression models, you can also indicate which models you want the autoML library to use, by indicating from the list below. \n",
+    "## By default, all models will be used in the autoML training. \n",
+    "## If you want to change the metric that the regression models will use to optimize for, you can select one metric from regression_metrics below. \n",
+    "## The default metric is R2.\n",
+    "\n",
+    "regression_models = [\n",
+    "    \"lr\",\n",
+    "    \"lasso\",\n",
+    "    \"ridge\",\n",
+    "    \"en\",\n",
+    "    \"lar\",\n",
+    "    \"llar\",\n",
+    "    \"omp\",\n",
+    "    \"br\",\n",
+    "    \"ard\",\n",
+    "    \"par\",\n",
+    "    \"ransac\",\n",
+    "    \"tr\",\n",
+    "    \"huber\",\n",
+    "    \"kr\",\n",
+    "    \"svm\",\n",
+    "    \"knn\",\n",
+    "    \"dt\",\n",
+    "    \"rf\",\n",
+    "    \"et\",\n",
+    "    \"ada\",\n",
+    "    \"gbr\",\n",
+    "    \"mlp\",\n",
+    "    \"xgboost\",\n",
+    "    \"lightgbm\",\n",
+    "    \"dummy\"\n",
+    "]\n",
+    "\n",
+    "regression_metrics = [\n",
+    "    \"mae\",\n",
+    "    \"mse\",\n",
+    "    \"rmse\",\n",
+    "    \"r2\",\n",
+    "    \"rmsle\",\n",
+    "    \"mape\"\n",
+    "]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First create a project on Gretel Cloud using the following example project name. Then, notice that the config includes both the synthetic data model and evaluation model. Note we're using the default Gretel ACTGAN model configuration in the following code."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a project with a name that describes this use case\n",
+    "from gretel_client.projects import create_or_get_unique_project\n",
+    "\n",
+    "project = create_or_get_unique_project(name=\"heart-disease-regression-notebook\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gretel_client.helpers import poll\n",
+    "from gretel_client.projects.models import read_model_config\n",
+    "\n",
+    "# We'll import the Dow Jones stock price dataset from Gretel's public S3 bucket\n",
```

Comment:
issue: comment does not match the dataset we are loading.

Reply:
ty for catching this